### ISSUE TITLE
Refactor the 

### ISSUE THREAD

**Issue by vrathi101** [PolicyEngine/policyengine-us#4736]
Thanks for filing an issue! Please remove any top-level headers that do not apply.

# Variable to add: name

# Variable to amend: name

# Parameter to add: name

# Parameter to amend: name

**Comment by MaxGhenis** [comment]
I think we need the special handling for this helper function. @nikhilwoodruff  created it and can advise. You could also take a look at the pr that created it.

**Comment by vrathi101** [comment]
Yeah, as I looked through the files that use this helper function, I also realized that, as each variable uses different earned and unearned incomes.
